### DISCUSSION TITLE
🔥 FrontendBR Space • Edição Front in Sampa

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#1124] [Dicas]
<img width="577" alt="screen shot 2018-06-25 at 4 15 14 pm" src="https://user-images.githubusercontent.com/3603793/41870707-310bf834-7893-11e8-868e-a44c2573d814.png">

Estavam curiosos para saber qual a [surpresa que preparamos](https://github.com/frontendbr/forum/issues/1117) para acontecer durante o Front in Sampa?

Nós teremos o **FrontendBR Space**! 😄 

Um espaço aberto e pensado para a comunidade INTEIRA relaxar e se divertir durante o evento.

**Quais as atrações?**

- Café e água 
- Sofás, poltronas e puffs
- Snacks
- Lives durante o evento
- Sorteios
- Aquela resenha bacana
- E principalmente... **MUITO NETWORKING**

O  **FrontendBR Space** vai acontecer no dia 14/07/2018, durante o Front in Sampa, dentro do [Hotel Maksoud Plaza](https://www.google.com.br/maps?q=Hotel+Maksoud+Plaza&um=1&ie=UTF-8&sa=X&ved=0ahUKEwj8oP7HxO_bAhXDnJAKHYIYAk8Q_AUICigB).

Esse é um presente do Front in Sampa e do Front-end BR para todos vocês e esperamos que todos aproveitem muito ❤️

**Comment by didiraja** [comment]
Muito bacana a ideia das lives, até a gente que ficou de fora vai poder curtir 😄
